abdm_python_integrator/abha/utils/abha_creation_util.py

```diff
@@ -0,0 +1,44 @@
+from abdm_python_integrator.abha.const import (
+    CREATE_HEALTH_ID_URL,
+    GENERATE_AADHAAR_OTP_URL,
+    GENERATE_MOBILE_OTP_URL,
+    VERIFY_AADHAAR_OTP_URL,
+    VERIFY_MOBILE_OTP_URL,
+)
+from abdm_python_integrator.utils import ABDMRequestHelper
+
+
+def generate_aadhaar_otp(aadhaar_number):
+    payload = {"aadhaar": str(aadhaar_number)}
```

Comment:
Should we pre-validate `aadhaar_number` and `mobile_number` to avoid unncessary bad requests?

Reply:
Yes, I think it is good to have this.

Reply:
Addressed in aa80126000e5eb7398f05531d919934e4893f94b